recipes/install_monitoring.rb

```diff
@@ -35,6 +33,45 @@
 node.set['cassandra']['install_java'] = false # The Abiquo jdk package is installed instead
 include_recipe 'cassandra-dse'
 
+service 'kairosdb' do
+  action :nothing
+end
+
+package 'kairosdb' do
+  action :install
+  # We want the service to be stopped immediately after the rpm package starts it, if we are going
+  # to replace its config file by a systemd unit. Otherwise we won't be able to properly manage it.
+  notifies :stop, 'service[kairosdb]', :immediately if node['platform_version'].to_i == 7
+end
+
+# Configure a systemd script if we are in CentOS 7
+file '/etc/init.d/kairosdb' do
+  action :delete
+  only_if { node['platform_version'].to_i == 7 }
+end
+
+systemd_unit 'kairosdb.service' do
+  content <<-EOU.gsub(/^\s+/, '')
+	[Unit]
+	Description=KairosDB is a fast distributed scalable time series database written on top of Cassandra.
+	Requires=cassandra.service
+	After=cassandra.service
```

Comment:
C* and kairos could be on different hosts. This unit should not depend on cassandra?